imports/startup/client/listenActiveUsers.js

```diff
@@ -0,0 +1,48 @@
+import { Meteor } from 'meteor/meteor';
+import { Tracker } from 'meteor/tracker';
+
+const updateUser = (user, force = false) => {
```

Comment:
This name is strange since we do not update if force is not true 😄

Reply:
:+1: